src/main/kotlin/app/revanced/patches/idaustria/patch/SpoofSignaturePatch.kt

```diff
@@ -0,0 +1,33 @@
+package app.revanced.patches.idaustria.patch
+
+import app.revanced.patcher.annotation.*
+import app.revanced.patcher.data.BytecodeContext
+import app.revanced.patcher.extensions.addInstructions
+import app.revanced.patcher.patch.BytecodePatch
+import app.revanced.patcher.patch.PatchResult
+import app.revanced.patcher.patch.PatchResultSuccess
+import app.revanced.patcher.patch.annotations.Patch
+import app.revanced.patches.idaustria.fingerprints.SpoofSignatureFingerprint
+
+@Patch
+@Name("spoof-signature")
+@Description("Spoof the correct app-fingerprint")
```

Comment:
Apply this change to the class name as well.

```suggestion
@Name("signature-spoof")
@Description("Spoofs the signature of the app.")
```